node/forcedroid.js

```diff
@@ -462,7 +462,17 @@ function fixSdkGradleFiles(config) {
 
 function fixSdkGradleFileHelper(appFolderName, lib) {
     console.log('Tweaking build.gradle for library ' + lib);
-    shelljs.sed('-i', "compile project(':libs:", "compile project(':forcedroid:libs:", path.join(appFolderName, "forcedroid", "libs", lib, "build.gradle"));    
+    replaceInFile(path.join(appFolderName, "forcedroid", "libs", lib, "build.gradle"), "compile project(':libs:", "compile project(':forcedroid:libs:"); 
+}
+
+function replaceInFile(fileName, textInFile, replacementText) {
+    var contents = fs.readFileSync(fileName, 'utf8');
+    var lines = contents.split(/\r*\n/);
+    var result = lines.map(function (line) {
+      return line.replace(textInFile, replacementText);
+    }).join('\n');
```

Comment:
Because we split the file contents on \r\n. If we don't join on \n the file contents will be on one line. The file becomes unreadable.